pages/services/edge-lb/1.2/pool-configuration/index.md

```diff
@@ -12,18 +12,20 @@ This is a reference for all possible Edge-LB pool configuration values, as well
 
 # Configuration Reference
 
-The configuration reference containing all possible options and descriptions of each can be found by running the following command:
+The API reference, in swagger format, containing all possible options and short descriptions of each endpoint can be found by running the following command:
 
 ```
 dcos edgelb show --reference
 ```
 
 For more information, see the [CLI Reference Guide entry for `dcos edgelb show`](/1.1/cli-reference/dcos-edgelb-show/).
 
+Choose an API version at the left to view the appropriate configuration reference or examples.
```

Comment:
I would rather say `on the sidebar` than on the left. It maybe the case that I am reading on a mobile, and then the collapsed responsive menu is not on the left before I have clicked the `hamburg` menu.